src/datasources/transaction-api/transaction-api.service.spec.ts

```diff
@@ -26,20 +40,17 @@ const configurationService = {
 } as jest.MockedObjectDeep<IConfigurationService>;
 const mockConfigurationService = jest.mocked(configurationService);
 
-const httpErrorFactory = {
-  from: jest.fn(),
-} as jest.MockedObjectDeep<HttpErrorFactory>;
-const mockHttpErrorFactory = jest.mocked(httpErrorFactory);
```

Comment:
I migrated to using the real HttpErrorFactory as we were mocking and testing the mocked functionality which didn't make much sense. Would welcome your thoughts regarding this.